### ISSUE TITLE
BBB recordings can not be deleted

### ISSUE THREAD

**Issue by NicoDucou** [chamilo/chamilo-lms#2826]
**Describe**
Deleteing a record of a BBB video conference does not work, it always gives an error message instead of the success message.

**To Reproduce**
Open a BBB videoconference and record it.
Once the conference is finished then wait for the recording to be available.
In the BBB tool in the list of BBB session in the action colomn click on the red cross to delete the recording
The error message appears on top : 
![image](https://user-images.githubusercontent.com/6579268/53740564-c43e2600-3e94-11e9-9297-fb20a58e3003.png)

Tested on  https://11.chamilo.org/plugin/bbb/listing.php?cidReq=NICODU&id_session=0&gidReq=0

Not that when there is no recording the meeting gets deleted without problem, so it's only when there is a call to BBB API that it generates an error.

**Expected behavior**
It should give the success message and the record should be deleted

**Server
 - Version of Chamilo: 1.11.x

**Comment by ywarnier** [comment]
I think @jmontoyaa is the most up-to-date with that plugin.

**Comment by ywarnier** [comment]
It would be nice to have it in 1.11.x but not absolutely mandatory

**Comment by ywarnier** [comment]
@NicoDucou Was this one fixed, maybe, with the other many fixes we sent to the BBB plugin in the meantime?

**Comment by NicoDucou** [comment]
> @NicoDucou Was this one fixed, maybe, with the other many fixes we sent to the BBB plugin in the meantime?

Yes, I just tested and it works fine now.
